test/controllers/api/rhsm/candlepin_proxies_controller_test.rb

```diff
@@ -123,6 +123,22 @@ module Katello
         assert_equal 200, response.status
       end
 
+      it "should fail with missing attribute 1" do
+        put :enabled_repos, params: { :id => @host.subscription_facet.uuid }
+        assert_equal 400, response.status
+      end
+
+      it "should fail with missing attribute 2" do
+        put :enabled_repos, params: { :id => @host.subscription_facet.uuid, :enabled_repos => {} }
+        assert_equal 400, response.status
+      end
+
+      it "should unbind all" do
+        Host::ContentFacet.any_instance.expects(:update_repositories_by_paths).with([])
+        put :enabled_repos, params: { :id => @host.subscription_facet.uuid, :enabled_repos => {"repos" => []}}
+        assert_equal 200, response.status
+      end
```

Comment:
@jturel Seems like the current version already handles that case...

Reply:
nice. thanks for adding the test coverage!

Reply:
The `dig` version might still be more specific. We should use that and keep the tests.